src/lib/edge-functions/deploy.mjs

```diff
@@ -8,17 +8,6 @@ import { EDGE_FUNCTIONS_FOLDER, PUBLIC_URL_PATH } from './consts.mjs'
 
 const distPath = getPathInProject([EDGE_FUNCTIONS_FOLDER])
 
-export const deployFileNormalizer = (rootDir, file) => {
```

Comment:
This logic was moved to the `normalizePath` function in `src/utils/deploy/util.mjs`.